packages/sdk/client-services/src/packlets/invitations/invitations-handler.ts

```diff
@@ -181,6 +185,13 @@ export class InvitationsHandler {
     };
 
     let swarmConnection: SwarmConnection;
+
+    // TODO(nf): cancel invitations when the persistence deadline is reached.
+    // TODO(nf): honor some deadline for non-persistent invitations as well?
```

Comment:
right -- re my comment to just have a field named "expiration" (and add a "created" TS).